tests/testthat/helper-daedalus-api.R

```diff
@@ -4,3 +4,23 @@ daedalus_api_endpoint <- function(..., validate = TRUE) {
     validate = validate
   )
 }
+
+skip_if_no_redis <- function() {
+  # Allows skipping redis/rrq e2e tests on mac os
```

Comment:
Pattern stolen from orderly.runner - docker isn't installed on mac os so in that case we don't attempt to run a redis container, or any of the tests which require it.